srcpkgs/llvm18/template

```diff
@@ -0,0 +1,725 @@
+
+	# Remove llvm-config-host in cross builds.
+	if [ "$CROSS_BUILD" ]; then
+		rm -f ${DESTDIR}/usr/bin/llvm-config-host
+	fi
+
+	rm -rf ${DESTDIR}/usr/share/gdb
+
+	# Install libcxxabi headers
+	vinstall ${wrksrc}/libcxxabi/include/__cxxabi_config.h 644 usr/include
+	vinstall ${wrksrc}/libcxxabi/include/cxxabi.h 644 usr/include
+
+	# Install libunwind headers
+	vinstall ${wrksrc}/libunwind/include/__libunwind_config.h 644 usr/include
+	vinstall ${wrksrc}/libunwind/include/libunwind.h 644 usr/include
+	vinstall ${wrksrc}/libunwind/include/unwind.h 644 usr/include
+	vinstall ${wrksrc}/libunwind/include/mach-o/compact_unwind_encoding.h \
+		644 usr/include/mach-o
+
+	# Install libcxx and libunwind docs
+	cmake -DCMAKE_INSTALL_PREFIX=${DESTDIR}/usr -P ${wrksrc}/${build_wrksrc}/runtimes-doc/libcxx/docs/cmake_install.cmake
+	cmake -DCMAKE_INSTALL_PREFIX=${DESTDIR}/usr -P ${wrksrc}/${build_wrksrc}/runtimes-doc/libunwind/docs/cmake_install.cmake
+
+	# Can this be disabled some other way?
+	rm -rf ${DESTDIR}/usr/lib64
+}
+
+clang18_package() {
+	lib32disabled=yes
+	depends="libstdc++-devel libgcc-devel  binutils ${XBPS_TARGET_LIBC}-devel
+	 compiler-rt18"
+	short_desc+=" - C language family frontend"
+	homepage="https://clang.llvm.org/"
+	pkg_install() {
+		vmove usr/bin/clang-18
+		vmove usr/bin/clang
+		vmove usr/bin/clang++
+		vmove usr/bin/clang-cl
+		vmove usr/bin/clang-cpp
+		vmove usr/bin/clang-check
+		vmove usr/bin/clang-extdef-mapping
+		vmove usr/bin/clang-format
+		vmove usr/bin/clang-linker-wrapper
+		vmove usr/bin/clang-offload-bundler
+		vmove usr/bin/clang-offload-packager
+		vmove usr/bin/clang-refactor
+		vmove usr/bin/clang-rename
+		vmove usr/bin/clang-repl
+		vmove usr/bin/clang-scan-deps
+		vmove usr/bin/clang-tblgen
+		vmove usr/bin/c-index-test
+		vmove usr/bin/diagtool
+		vmove usr/bin/amdgpu-arch
+		vmove usr/bin/nvptx-arch
+		vmove usr/share/man/man1/clang.1
+		vmove usr/share/man/man1/diagtool.1
+		if [ "$build_option_polly" ]; then
+			vmove usr/lib/LLVMPolly.so
+			vmove usr/share/man/man1/polly.1
+		fi
+	}
+}
+
+clang18-headers_package() {
+	lib32disabled=yes
+	short_desc+=" - C language family frontend - Headers"
+	homepage="https://clang.llvm.org/"
+	pkg_install() {
+		vmove usr/lib/clang/18/include
+	}
+}
+
+clang18-devel_package() {
+	lib32disabled=yes
+	depends="libstdc++-devel libgcc-devel  binutils ${XBPS_TARGET_LIBC}-devel
+	 clang18>=${version}_${revision} clang-analyzer18>=${version}_${revision}
+	 llvm18>=${version}_${revision}"
+	if [ "$build_option_clang_tools_extra" ]; then
+		depends+=" clang-tools-extra18>=${version}_${revision}"
+	fi
+	short_desc+=" - C language family frontend - development Files"
+	homepage="https://clang.llvm.org/"
+	pkg_install() {
+		vmove usr/include/clang
+		vmove usr/include/clang-c
+		vmove usr/lib/libear
+		vmove usr/lib/cmake/clang
+		vmove "usr/lib/libclang*.a"
+		vmove "usr/lib/libclang*.so"
+		vmove usr/share/clang
+		vmove usr/bin/hmaptool
+		vmove usr/bin/git-clang-format
```

Comment:
`hmaptool` goes to `-tools-extra`, `git-clang-format` goes to same packages with `clang-format`.

Reply:
Honestly, I think `hmaptool` should go to clang